File: study-notes/flutter/04-Dart的面向对象/01_类的定义.dart
main(List<String> args) {
  var p = Person("coderZsq", 18);
}

class Person {
  String name;
  int age;

  // Person(String name, int age) {
  //   this.name = name;
  //   this.age = age;
  // }

  Person(this.name, this.age);
}

File: study-notes/subjects/vue/08-TypeScript/09_TypeScript中类的使用/01_类的定义.ts
class Person {
  name: string;
  age: number;

  constructor(name: string, age: number) {
    this.name = name;
    this.age = age;
  }

  eating() {
    console.log(this.name + ' eating');
  }
}

const p = new Person('sq', 18);
console.log(p.name);
console.log(p.age);
p.eating();

export {};
